R/RDataParam.R

```diff
@@ -0,0 +1,70 @@
+#' @title Store `XcmsExperiment` object as .RData file
+#'
+#' @name RDataParam
+#'
+#' @export
+#'
+#' @description
+#' The `RDataParam` class and method allow users to save an `XcmsExperiment`
+#' object as an .RData file with a chosen filename. This new `param` class and
+#' method are part of the possible dispatch of the generic function
+#' `storeResults`. 
+#' 
+#' Other available `param` classes and linked methods include:
+#'
+#' - `PlainTextParam`: ...
+#'
+#' - `MzTabMParam`: ...
+#' 
+#' @param param A parameter defining the format in which the object should be
+#' saved. For the `RDataParam`, it has one slot for the `fileName` in which the
+#' object is going to be saved. The default will be `tempfile()`.
+#'
+#' @param object An object of class `XcmsExperiment` that will be saved as an
+#' .RData file.
+#' 
+#' @return The saved object as an .RData file. 
+#'
+#' @author Philippine Louail
+#'
+#' @examples
+#'
+#'## Get an `XcmsExperiment` object
+#' x <- ...
```

Comment:
Not sure where to get an XcmsExperiment object, should I do something similar to the testthat.R file ? But then I would need to load packages and i don't know if that's okay.

Reply:
you can use this:

```
#' ## Load a test data set with detected peaks
#' faahko_sub <- loadXcmsData("faahko_sub2")

```